invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/DocumentList/DocumentStats.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Header, List} from "semantic-ui-react";
+import {Link} from "react-router-dom";
+
+export default class DocumentStats extends Component {
+  render() {
+    const {metadata} = this.props;
+    return (
+      <List verticalAlign="middle" style={{color: '#aaa'}}>
```

Comment:
Should probably move this (and the color below) to the theme instead.

Reply:
thanks, I moved the code around for now, I will remove the styling once I will arrive to styling that page